test/api/apikeys_test.go

```diff
@@ -124,20 +121,12 @@ func testPrivateAPIKeysUsage(t *testing.T, at *test.AccountsTester) {
 	// Get user stats without a cookie or headers - pass the API key via a query
 	// variable. The main thing we want to see here is whether we get
 	// an `Unauthorized` error or not but we'll validate the stats as well.
-	params := url.Values{}
-	params.Set("apiKey", akWithKey.Key.String())
-	_, body, err := at.Get("/user/stats", params)
-	if err != nil {
-		t.Fatal(err, string(body))
-	}
-	var us database.UserStats
-	err = json.Unmarshal(body, &us)
-	if err != nil {
-		t.Fatal(err)
+	h := map[string]string{
+		api.APIKeyHeader: akWithKey.Key.String(),
 	}
-	if us.TotalUploadsSize != uploadSize || us.NumUploads != 1 || us.BandwidthUploads != skynet.BandwidthUploadCost(uploadSize) {
-		t.Fatalf("Unexpected user stats. Expected TotalUploadSize %d (got %d), NumUploads 1 (got %d), BandwidthUploads %d (got %d).",
-			uploadSize, us.TotalDownloadsSize, us.NumUploads, skynet.BandwidthUploadCost(uploadSize), us.BandwidthUploads)
+	ulg, _, err := at.UserLimits("", h)
```

Comment:
Seems like you are not validating the error here?

Reply:
Added.